translator/totomlconfig/toTomlConfig_test.go

```diff
@@ -202,3 +219,21 @@ func resetContext() {
 
 	os.Setenv("ProgramData", "c:\\ProgramData")
 }
+
```

Comment:
Could we add a comment about how the two tomls are being evaluated into the go object? This may also be useful when someone needs to add or extend the toToml.go file.